app/controllers/admin/sessions_controller.rb

```diff
@@ -0,0 +1,8 @@
+# Namespaced Admin class
```

Comment:
You probably don't need to specify this - let's assume people reading this code understand some Ruby